osgar/drivers/realsense.py

```diff
@@ -148,9 +163,14 @@ def start(self):
         if enable_depth:
             self.depth_pipeline = rs.pipeline(ctx)
             depth_cfg = rs.config()
-            depth_cfg.enable_stream(rs.stream.depth, 640, 360, rs.format.z16, 30)
+            w, h = self.default_depth_resolution
```

Comment:
Do we really want to change the default resolution from 640x360 to 640x480? I remember there being some peculiarities, though I don't remember the details. You might want to check if adding more lines does not decrease the horizontal FOV. According to https://www.intel.com/content/dam/support/us/en/documents/emerging-technologies/intel-realsense-technology/BKMs_Tuning_RealSense_D4xx_Cam.pdf the picked default resolution 640x360 should have similar aspect ratio to the native resolution 848x480 which should optimally use the available sensor size and optics.

Reply:
Thanks for notice. So far I have returned the resolution back.